### DISCUSSION TITLE
1.3.1版本不支持STRING（字符串）吗？哪个版本开始支持？

### DISCUSSION THREAD

**Discussion by jibite2050** [apache/iotdb#12819] [Q&A] [answered]
技术文档描述支持String类型，
![微信截图_20240627134607](https://github.com/apache/iotdb/assets/142650458/16370439-4ead-4c57-b410-09a8bc1cd6af)
使用提示不支持。
![ca0b71efed4ddb427acbe4f31cb3012f](https://github.com/apache/iotdb/assets/142650458/c4974b0c-4f35-4b8c-809d-d3f8bd4b0d4c)

从哪个版本开始支持String类型呢？

**Comment by HTHou** [comment, marked as answer]
这个类型会在 1.3.3 版本中支持

**Reply by wenmin864** [reply]
大概计划什么时候会发布这个版本

**Comment by JavaCPlusMario** [comment]
用 text 不香 嘛
